source/core/parameters.cc

```diff
@@ -49,6 +49,20 @@ DeclException3(MultipleAdaptationSizeError,
                << arg2 << ") does not correspond to the number of 'variables' ("
                << arg3 << ")");
 
+DeclException1(
+  PorousMediaDisabledError,
+  double,
+  << "You specified a penetration depth of " << arg1
+  << ". However, the 'type' parameter is not set to 'exponential decay'.");
```

Comment:
Is that necessary? I mean you can have a value of the parameter and not use the model, would that really pose au problem?

Reply:
It was more to avoid the user mistake of thinking that they were changing something by changing the ``penetration depth`` while using the ``material_interface_vof`` laser model... to avoid using mistakenly the wrong model. However, if you feel it is unnecessary, I could take it out.